tibanna_ffcommon/portal_utils.py

```diff
@@ -214,7 +214,7 @@ def add_args(self, ff_meta):
         # create args
         args = dict()
         for k in ['app_name', 'app_version', 'cwl_directory_url', 'cwl_main_filename', 'cwl_child_filenames',
-                  'wdl_directory_url', 'wdl_main_filename', 'wdl_child_filenames', 'workflow_engine']:
+                  'wdl_directory_url', 'wdl_main_filename', 'wdl_child_filenames', 'workflow_engine', 'run_args']:
```

Comment:
Tibanna support `run_args`. This will make sure that tibanna_ff passes it on to Tibanna (needed for the chip-seq pipeline).